### ISSUE TITLE
Preview URL 301 redirect

### ISSUE THREAD

**Issue by kestraly** [bentideswell/magento2-wordpress-integration#118]
We've recentlly updated the connector, however it appears to now be 301 redirecting preview links to the blog base

On closer inspection it appears to be due to the ending /

https://www.domain.com/blog/?p=3313&preview=true&preview_time=1666101335 <!-- 301 redirections to https://www.domain.com/blog

https://www.domain.com/blog?p=3313&preview=true&preview_time=1666101335 <!-- Preview is correct

It might be related to another module cropping trailing slashes. We're still investigating at this phase, but wanted to flag because this is an issue for stores that do crop trailing slashes

**Comment by bentideswell** [comment]
To fix trailing slash issues, ensure that the trailing slash configuration is the same in Magento and WordPress. You should change WordPress to use the same configuration as Magento.

WordPress determines whether to use a trailing slash based on the permalink structure at WP Admin > Settings > Permalinks. If the URL structure here has a trailing slash then all WP URLs will.

From the look of it, your Magento site does not use a trailing slash so make the permalink structure in WordPress also does not use a trailing slash.

**Comment by bentideswell** [comment]
Great, thanks for confirming. I think this is resolved so I will close the issue, but if the issue is still open or comes back, feel free to re-open and comment again.

**Comment by kestraly** [comment]
Sort of resolved. Doesn't work when I remove the / again. 

I'm just going to leave the ending / on for the moment so the previews work

**Comment by bentideswell** [comment]
Could Magento be redirecting the URLs? Do all URLs with trailing slashes redirect?

**Comment by kestraly** [comment]
Yeah, all other URL's remove the trailing slash. It's being used in conjunction with Mageworx SEO Suite, so I suspect it is something to do with that. 

If I turn on trailing slash on wordpress, it remains on the blog section and isn't removed, so that is particularly strange situation if it's being affected for previews, but not if it's intentionally being added.
